### PR TITLE
v0.6.5

### PR DESCRIPTION
**Pull request by irlaec**

### CODE REVIEW

File: octoprint_mrbeam/static/js/working_area.js [hunk 1]
```diff
@@ -2664,6 +2688,84 @@ $(function(){
 		//  QUICKTEXT end
 		// ***********************************************************
 
+		// ***********************************************************
+		//  QUICKSCAN start
+		// ***********************************************************
+
```
Comment (amrsoll) [inline octoprint_mrbeam/static/js/working_area.js]: Is quickscan ready for Beta @Gallore ?

**Comment by amrsoll** [comment]
- Removes the dependency on the legacy algorithm (and a bg associated with it)
- Adds some better install process for the desktop

File: octoprint_mrbeam/static/js/working_area.js (hunk 1 above)
Comment (andywerner) [reply octoprint_mrbeam/static/js/working_area.js]: nope!

**Review by andywerner — CHANGES_REQUESTED** [review]
Quick scan must be hidden! It's not in a beta state.

**Review by amrsoll — APPROVED** [review]
Everything is fixed now :)

**Review by andywerner — APPROVED** [review]

**Pull request merged**